### ISSUE TITLE
update nodejs

### ISSUE THREAD

**Issue by staabm** [FriendsOfREDAXO/installer-action#108]
beim verwenden der action erscheint eine warning

```
Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: actions/checkout@v2
```

siehe https://github.com/FriendsOfREDAXO/rexstan/actions/runs/3443853549

**Comment by bloep** [comment]
das ist kein direktes Problem der Action sondern des Workflows, weil dort noch die `actions/checkout@v2` vertreten ist.

**Comment by bloep** [comment]
ich würde dennoch zumindest bei FOR mal alle Repos durchgehen und die Workflows updaten
https://github.com/FriendsOfREDAXO/installer-action/network/dependents?owner=FriendsOfREDAXO&dependent_type=REPOSITORY&owner=FriendsOfREDAXO

**Comment by staabm** [comment]
Danke

**Comment by bloep** [comment]
alle `checkout@v2` vorkommen in workflows.
https://github.com/search?q=org%3AFriendsOfREDAXO+%22checkout%40v2%22+language%3AYAML&type=code&l=YAML

noch offene Workflows (nicht archivierte Repos) mit `checkout@v2`:
- [ ] https://github.com/FriendsOfREDAXO/collaborate/blob/main/.github/workflows/publish-to-redaxo.yml
- [ ] https://github.com/FriendsOfREDAXO/license/blob/main/.github/workflows/publish-to-redaxo-org.yml
- [ ] https://github.com/FriendsOfREDAXO/content/blob/main/.github/workflows/publish-to-redaxo.yml
- [ ] https://github.com/FriendsOfREDAXO/Modulsammlung/blob/master/.github/workflows/publish-to-redaxo-org.yml
- [ ] https://github.com/FriendsOfREDAXO/media_manager_plus/blob/master/.github/workflows/publish-to-redaxo-org.yml
- [ ] https://github.com/FriendsOfREDAXO/transient/blob/master/.github/workflows/publish-to-redaxo-org.yml
- [ ] https://github.com/FriendsOfREDAXO/multiglossar/blob/master/.github/workflows/publish-to-redaxo.yml
- [ ] https://github.com/FriendsOfREDAXO/textile_migration/blob/master/.github/workflows/publish-to-redaxo-org.yml
- [ ] https://github.com/FriendsOfREDAXO/yform_standalone_validator/blob/master/.github/workflows/publish-to-redaxo-org.yml
- [ ] https://github.com/FriendsOfREDAXO/ffmpeg/blob/master/.github/workflows/publish-to-redaxo-org.yml
- [ ] https://github.com/FriendsOfREDAXO/project_manager/blob/main/.github/workflows/publish-to-redaxo-org.yml
- [ ] https://github.com/FriendsOfREDAXO/yform_export/blob/master/.github/workflows/publish-to-redaxo-org.yml
- [ ] https://github.com/FriendsOfREDAXO/hyphenator/blob/master/.github/workflows/publish-to-redaxo-org.yml
- [ ] https://github.com/FriendsOfREDAXO/article_presets/blob/master/.github/workflows/publish-to-redaxo-org.yml
- [ ] https://github.com/FriendsOfREDAXO/be_password/blob/master/.github/workflows/publish-to-redaxo.yml
- [ ] https://github.com/FriendsOfREDAXO/navbuilder/blob/master/.github/workflows/publish-to-redaxo-org.yml
- [ ] https://github.com/FriendsOfREDAXO/zip_install/blob/master/.github/workflows/publish-to-redaxo-org.yml
- [ ] https://github.com/FriendsOfREDAXO/mediapool_exif/blob/master/.github/workflows/publish-to-redaxo.yml
- [ ] https://github.com/FriendsOfREDAXO/yform_mediafile/blob/main/.github/workflows/publish-to-redaxo.yml
- [ ] https://github.com/FriendsOfREDAXO/yform_geo_osm/blob/master/.github/workflows/publish-to-redaxo.yml
- [ ] https://github.com/FriendsOfREDAXO/fa_iconpicker/blob/main/.github/workflows/publish-to-redaxo.yml
- [ ] https://github.com/FriendsOfREDAXO/minify_images/blob/master/.github/workflows/publish-to-redaxo.yml
- [ ] https://github.com/FriendsOfREDAXO/cropper/blob/master/.github/workflows/publish-to-redaxo.yml
- [ ] https://github.com/FriendsOfREDAXO/slice_columns/blob/main/.github/workflows/publish-to-redaxo.yml
- [ ] https://github.com/FriendsOfREDAXO/media_srcset/blob/master/.github/workflows/publish-to-redaxo.yml
- [ ] https://github.com/FriendsOfREDAXO/ui_tools/blob/master/.github/workflows/publish-to-redaxo.yml
- [ ] https://github.com/FriendsOfREDAXO/markitup/blob/master/.github/workflows/publish-to-redaxo.yml
- [ ] https://github.com/FriendsOfREDAXO/login_neon/blob/main/.github/workflows/publish-to-redaxo.yml
- [ ] https://github.com/FriendsOfREDAXO/redactor2/blob/master/.github/workflows/publish-to-redaxo.yml
- [ ] https://github.com/FriendsOfREDAXO/uploader/blob/master/.github/workflows/publish-to-redaxo.yml
- [ ] https://github.com/FriendsOfREDAXO/cache_warmup/blob/main/.github/workflows/publish-to-redaxo.yml
- [ ] https://github.com/FriendsOfREDAXO/yrewrite_scheme/blob/master/.github/workflows/publish-to-redaxo.yml
- [ ] https://github.com/FriendsOfREDAXO/tinymce5/blob/master/.github/workflows/publish-to-redaxo.yml
- [ ] https://github.com/FriendsOfREDAXO/redactor/blob/master/.github/workflows/publish-to-redaxo.yml
- [ ] https://github.com/FriendsOfREDAXO/minibar/blob/master/.github/workflows/publish-to-redaxo.yml

**Comment by staabm** [comment]
vllt können wir künftig den workflow als komponente aus diesem repo hier laden statt überall hinzukopieren?

**Comment by TobiasKrais** [comment]
Ich habe folgende Warnung. Ich verwende actions/checkout@v4.

redaxo_publish
Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: FriendsOfREDAXO/installer-action@v1. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.

**Comment by alxndr-w** [comment]
#282 wie geht es hier weiter? Was ist zu tun?

**Comment by alxndr-w** [comment]
`actions/checkout@v4` @TobiasKrais wirklich v4?

**Comment by TobiasKrais** [comment]
Ja. Hier der Workflow: https://github.com/FriendsOfREDAXO/multinewsletter/blob/7e93cd18fee622594490246cc8f942acf988305a/.github/workflows/publish-to-redaxo-org.yml#L13 und der letzte run der Action: https://github.com/FriendsOfREDAXO/multinewsletter/actions/runs/10487171491
